backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -122,11 +122,24 @@ public IActionResult UpdateUser([FromBody] Model.UserModel model)
         [HttpDelete]
         public IActionResult DeleteUser([FromBody] Model.UserModel model)
         {
-            var entity = _mapper.Map<Entities.User>(model);
+            var entity = _mapper.Map<Entity.User>(model);
             _pimsAdminService.User.Remove(entity);
 
             return new JsonResult(model);
         }
+
+        /// <summary>
+        /// Gets all of the access requests that have been submitted to the system.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("/api/accessRequests")]
+        public IActionResult GetAccessRequests(int page = 1, int quantity = 10, string sort = null, bool? isGranted = null)
+        {
+            Entity.Models.Paged<Entity.AccessRequest> result = _pimsAdminService.User.GetAccessRequestsNoTracking(page, quantity, sort, isGranted);
```

Comment:
Would love to see a one-liner that gives us the paged results in the correct types (model).  Not this PR.